client/client_test.go

```diff
@@ -14,7 +15,11 @@ func TestMigrationSQLServers(t *testing.T) {
 	if err != nil {
 		log.Fatalf("Could not connect to docker: %s", err)
 	}
-	c, err := client.New("./testdata/config.yml", "", "")
+	cfg, err := config.Parse("./testdata/config.yml")
+	if err != nil {
+		t.Fatal(err)
+	}
+	c, err := client.New("", "", cfg)
```

Comment:
This is how users of the library now have to initialize a client. First they create a config object, then pass it into `client.New`